pkgs/development/libraries/pixman/default.nix

```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
 
   configureFlags = stdenv.lib.optional stdenv.isArm "--disable-arm-iwmmxt";
 
-  doCheck = true;
+  doCheck = stdenv.hostPlatform == stdenv.buildPlatform;
```

Comment:
Again stdenv should be changed to cope with this.

Reply:
Should it? It's not clear to me that `stdenv` should dictate this.